frontend/src/components/eMIB/constants.js

```diff
@@ -55,3 +55,10 @@ export const contactShape = PropTypes.shape({
   name: PropTypes.string.isRequired,
   role: PropTypes.string.isRequired
 });
+
+// The structure of a node in a treeview
```

Comment:
This is only being used in one file, so you can define this shape in the component itself.

Reply:
moved in 7896385